<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQQ</class>
 <widget class="QWidget" name="MyQQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>739</width>
    <height>499</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>499</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MyQQ</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="weiCHatResource.qrc">:/new/image/images/11.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>250</width>
     <height>65</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="weiCHatResource.qrc">:/new/image/images/22.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>0</y>
     <width>471</width>
     <height>65</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="weiCHatResource.qrc">:/new/image/images/33.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>1</y>
     <width>121</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>365</y>
     <width>432</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="weiCHatResource.qrc">:/new/image/images/44.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="searchPushButton">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>25</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="userListTableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>65</y>
     <width>250</width>
     <height>435</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTextBrowser" name="chatTextBrowser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>65</y>
     <width>431</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatTextEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>403</y>
     <width>431</width>
     <height>97</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="transPushButton">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>375</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sendPushButton">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>476</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发 送(S)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="weiCHatResource.qrc"/>
 </resources>
 <connections/>
</ui>
